tutorials/quality_standards.ipynb

```diff
@@ -0,0 +1,654 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Quality standards\n",
+    "\n",
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# if the package is not installed in your python environment, run this to execute the notebook directly from inside the GitHub repository\n",
+    "%cd -q .."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import asdf\n",
+    "import fs\n",
+    "import yaml\n",
+    "\n",
+    "from weldx.asdf.file import WeldxFile\n",
+    "from weldx.config import add_quality_standard, enable_quality_standard, QualityStandard\n",
+    "from weldx.measurement import  GenericEquipment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "The main purposes of WelDX is to provide a file format for experimental data that is self explanatory and contains all\n",
+    "relevant information about the conducted experiment.\n",
+    "This allows fellow researchers to understand and analyze the data without taking part in the experiment or having access\n",
+    "to the persons who did.\n",
+    "We assure this by introducing quality standards for each type of experiment and every other information that\n",
+    "describes certain aspects of an experiment like coordinate systems, sensors or specimens.\n",
+    "Every time a WelDX file is written or read, it is validated against those standards.\n",
+    "\n",
+    "## Schema files\n",
+    "\n",
+    "Every standard used by WelDX is defined by schema files.\n",
+    "A schema file is a [YAML](https://yaml.org/)-file that defines the allowed properties of an serializable object.\n",
+    "How such a schema file is generated is not part of this tutorial, but you can find further information regarding this\n",
+    "topic in the official\n",
+    "[documentation of the ASDF standard](https://asdf-standard.readthedocs.io/en/latest/schemas.html) which is the\n",
+    "foundation of a WelDX file.\n",
+    "Furthermore, an overview of all pre-defined schema files can be found on the [Standards](../standard.rst) page of this\n",
+    "documentation.\n",
+    "\n",
+    "> todo: fix link\n",
```

Comment:
which link should be fixed?

Reply:
Guess I was referring to the link to the standards. It's been a while since I wrote the tutorial. The problem is that the link doesn't work if you run the tutorial in a notebook, because the Kernel's working directory is (at least on my system) `weldx/tutorials`. In the documentation, the tutorials run in `weldx/doc/tutorials` because they are copied to this location.
So the relative paths to the corresponding `.rst` file differ. Ideas?